src/pint/models/parameter.py

```diff
@@ -437,30 +441,42 @@ def from_parfile_line_regular(self, line):
 
             if len(k) >= 4:
                 ucty = k[3]
-            self.uncertainty = self.set_uncertainty(ucty)
+            self.uncertainty = self._set_uncertainty(ucty)
         return True
 
+    def add_alias(self, alias):
+        """Add a name to the list of aliases for this parameter."""
+        self.aliases.append(alias)
+
     def name_matches(self, name):
         """Whether or not the parameter name matches the provided name"""
         return (name == self.name.upper()) or (
             name in map(lambda x: x.upper(), self.aliases)
```

Comment:
Might that be worth changing to a list comp?  Those seem way more readable to me than mapping with lambdas....

Reply:
Indeed.